tasks.py

```diff
@@ -742,7 +742,7 @@ def helmdocs(ctx, env="container"):
     "external_containers": "a comma separated list of external containers names to use for the test. (valid parameters are: ibgp-single-hop / ibgp-multi-hop / ebgp-single-hop / ebgp-multi-hop)",
     "native_bgp": "tells if the given cluster is deployed using native bgp mode ",
```

Comment:
Please, add some documentation for this new parameter:

```suggestion
    "native_bgp": "tells if the given cluster is deployed using native bgp mode ",
    "frr_image": "specifies which FRR container should be used in e2e tests when creating external router endpoints. Useful to test compatibility with multiple FRR versions.",
```

I'd also change it to `external_containers_frr_image_for`, as it doesn't change in any way the FRR containers MetalLB uses for its work. Right?

Reply:
right

Reply:
done